lib/spree_avatax_certified/engine.rb

```diff
@@ -29,10 +33,33 @@ class Engine < Rails::Engine
       end
     end
 
+    def self.configure_caching
+      SpreeAvataxCertified.cache = if ENV['MEMCACHED_SERVERS'].blank?
```

Comment:
since this yaml file is non standard and a quirk of EY. I'll parse it in the main app before the gem is activated. ie. the gem here won't change much, if at all.